charms/kfp-api/tests/integration/test_charm.py

```diff
@@ -61,23 +61,29 @@ async def test_build_and_deploy(ops_test: OpsTest):
 
 async def test_prometheus_grafana_integration(ops_test: OpsTest):
     """Deploy prometheus, grafana and required relations, then test the metrics."""
+    application_name = APP_NAME
```

Comment:
Do we need `application_name` if we can just use `APP_NAME`? Later in this test, we use `APP_NAME` anyway for testing the prometheus deployment

Reply:
ahh I missed that it was used more.  I localised it because I copied/pasted some code from elsewhere that took `application_name` as an arg, but I'll change it here.